flink-batch-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormatTest.java

```diff
@@ -58,54 +52,39 @@ public static void setUpClass() {
 
 	private static void prepareDerbyDatabase() throws ClassNotFoundException, SQLException {
 		System.setProperty("derby.stream.error.field", "org.apache.flink.api.java.io.jdbc.DerbyUtil.DEV_NULL");
-		String dbURL = "jdbc:derby:memory:ebookshop;create=true";
-		Class.forName("org.apache.derby.jdbc.EmbeddedDriver");
-		conn = DriverManager.getConnection(dbURL);
+		Class.forName(JDBCExample.DRIVER_CLASS);
+		conn = DriverManager.getConnection(JDBCExample.DB_URL+";create=true");
 		createTable();
 		insertDataToSQLTable();
 		conn.close();
 	}
 
 	private static void createTable() throws SQLException {
-		StringBuilder sqlQueryBuilder = new StringBuilder("CREATE TABLE books (");
-		sqlQueryBuilder.append("id INT NOT NULL DEFAULT 0,");
-		sqlQueryBuilder.append("title VARCHAR(50) DEFAULT NULL,");
-		sqlQueryBuilder.append("author VARCHAR(50) DEFAULT NULL,");
-		sqlQueryBuilder.append("price FLOAT DEFAULT NULL,");
-		sqlQueryBuilder.append("qty INT DEFAULT NULL,");
-		sqlQueryBuilder.append("PRIMARY KEY (id))");
-
 		Statement stat = conn.createStatement();
-		stat.executeUpdate(sqlQueryBuilder.toString());
+		stat.executeUpdate(JDBCExample.getCreateQuery());
```

Comment:
maybe I'm overthinking this, but in my opinion a change to an example shouldn't have an effect on a test. that's why I'd rather keep them separated.